internal/client-db-connector/connector.go

```diff
@@ -1,4 +1,177 @@
 package client_db_connector
 
-type Connector interface {
+import (
+	"context"
+	"fmt"
+	"github.com/ydb-platform/ydb-go-genproto/Ydb_Export_V1"
+	"github.com/ydb-platform/ydb-go-genproto/Ydb_Import_V1"
+	"github.com/ydb-platform/ydb-go-genproto/Ydb_Operation_V1"
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb"
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb_Export"
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb_Import"
+	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb_Operations"
+	"github.com/ydb-platform/ydb-go-sdk/v3"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/durationpb"
+	"time"
+	"ydbcp/internal/types"
+	"ydbcp/internal/util/xlog"
+)
+
+type ClientDbConnector interface {
+	ExportToS3(ctx context.Context, settings types.ExportToS3Settings) (string, error)
+	ImportFromS3(ctx context.Context, settings types.ImportFromS3Settings) (string, error)
+	GetOperationStatus(ctx context.Context, clientDb types.YdbConnectionParams, operationId string) (types.YdbOperationInfo, error)
+}
+
+type ClientDbConnectorImpl struct {
+}
+
+func (d *ClientDbConnectorImpl) ExportToS3(ctx context.Context, settings types.ExportToS3Settings) (string, error) {
+	clientDbConnectionString := settings.ClientDb.Endpoint + settings.ClientDb.DatabaseName
+
+	xlog.Info(ctx, "Connecting to client db", zap.String("dsn", clientDbConnectionString))
+	db, connErr := ydb.Open(ctx, clientDbConnectionString, ydb.WithAnonymousCredentials())
```

Comment:
we don't want to create ydb driver for each export, move it to the ClientDbConnectorImpl